src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1591941085.inc.php

```diff
@@ -0,0 +1,80 @@
+<h1>Build #1591941085</h1>
+<h2>Date: 2020-06-12</h2>
+<div class="changelog">
+    - #478: (related) Show menu entry for table; add new field type for this
+</div>
+<?php
+
+$fieldTypeId = TCMSLogChange::createUnusedRecordId('cms_field_type');
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_type', 'en')
+    ->setFields([
+        '049_trans' => 'Sidebar menu entry lookup',
+        'force_auto_increment' => '0',
+        'constname' => 'CMSFIELD_SIDEBAR_LOOKUP',
+        'mysql_type' => '',
+        'length_set' => '',
+        'base_type' => 'standard',
+        'help_text' => '',
+        'mysql_standard_value' => '',
+        'fieldclass' => 'ChameleonSystem\CoreBundle\Field\FieldSidebarConnected',
+        'contains_images' => '0',
+        'indextype' => 'none',
+        'class_subtype' => '',
+        'class_type' => 'Core',
+        'id' => $fieldTypeId,
+    ])
+;
+TCMSLogChange::insert(__LINE__, $data);
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_type', 'de')
+    ->setFields([
+        'translation' => 'Menü-Eintrag Lookup',
+    ])
+    ->setWhereEquals([
+        'id' => $fieldTypeId,
+    ])
+;
+TCMSLogChange::update(__LINE__, $data);
+
+$fieldConfId = TCMSLogChange::createUnusedRecordId('cms_field_conf');
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
+  ->setFields([
+      'name' => 'shown_in_menu_as',
+      'translation' => 'Found in the menu as',
+      'cms_field_type_id' => TCMSLogChange::GetFieldType('CMSFIELD_SIDEBAR_LOOKUP'),
+      'cms_tbl_field_tab' => '',
+      'isrequired' => '0',
+      'fieldclass' => 'ChameleonSystem\CoreBundle\Field\FieldSidebarConnected',
+      'modifier' => 'readonly',
+      'field_default_value' => '',
+      'length_set' => '',
+      'fieldtype_config' => '',
+      'restrict_to_groups' => '0',
+      'field_width' => '0',
+      'position' => '2182',
+      '049_helptext' => 'Shows the first location if any where this table is accessible in the sidebar menu.',
```

Comment:
We currently assign a table to a menu item by editing the menu item. now we are introducing a connection from table to menu item... That makes it more complicated to add a table and organize the menu. It will also be difficult for the user to understand. why do we need this?

Reply:
It's also only for information purposes (read-only).